resources/profile.rb

```diff
@@ -1,2 +1,10 @@
 actions :create, :enable, :disable, :default
 default_action :create
+
+# man 'tuned.conf'
+# MAIN SECTION
+#   The main section is called "[main]"
+property :main, kind_of: Hash, default: {}
```

Comment:
your default value will be shared by every resource instance, you should at least freeze the empty hash